packages/math/src/bn.ts

```diff
@@ -91,6 +91,10 @@ export class BN extends BnJs implements BNInputOverrides, BNHiddenTypes, BNHelpe
     return this.toString(16);
   }
 
+  valueOf(): string {
+    return this.toString();
+  }
+
```

Comment:
not sure to hex this.  `bn` doesnt have value of and Bignumber does [this](https://mikemcl.github.io/bignumber.js/#valueOf)